libs/language-server/src/stdlib/domain/material-science/MatSciDoiStandardizer.jv

```diff
@@ -0,0 +1,82 @@
+// SPDX-FileCopyrightText: 2024 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+/** Transformer to standardize DOI references and remove dx.doi.org */
+publish transform RemoveDxDotDoiDotOrg {
+	from valueWithBraces oftype text;
```

Comment:
Input/output names in all transforms seem wrong (probably c&p)? Should match what the transform actually does, e.g. `from unstandardizedValue`, `to standardizedValue`.